app/src/main/res/layout/browser_overlay.xml

```diff
@@ -72,6 +84,25 @@
             android:lines="1"
             android:selectAllOnFocus="true"
             android:importantForAutofill="no"/>
+
+        <View
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_weight="1" />
+
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_weight="0"
+            android:tint="@android:color/white"
+            android:src="@drawable/ic_wide_menu" />
+
+        <TextView
```

Comment:
perf nit: you can remove the ImageView by adding a compound drawable directly into the TextView with `android:drawableStart` (or `compoundDrawable` whatever in the code). This is slightly better for performance, though it's harder to style the image as you want. If changing this starts giving you trouble, it's really not worth the time.